packages/core/.eslintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["../../.eslintrc.package.json"]
+  "extends": ["../../.eslintrc.package.json"],
+  "globals": {
+    "BUNDLE_VERSION":"readonly"
```

Comment:
just a nit, can you add a space after the `:`?

Reply:
okie thank you for catching it.